CHANGELOG.md

```diff
@@ -1,4 +1,21 @@
 # Changelog
+
+### Release 2.2.2 (August 19, 2019)
+
+[Milestone#36](https://github.com/awslabs/amazon-kinesis-client/milestone/36)
+* Fix to prevent invalid ShardConsumer state transitions due to rejected executor service executions
+  * [PR#560](https://github.com/awslabs/amazon-kinesis-client/pull/560)
+* Fixing bug where initial subscription failure causes shard consumer to get stuck
+  * [PR#562](https://github.com/awslabs/amazon-kinesis-client/pull/562)
+* Making CW publish failures visible
```

Comment:
We want the changelog text to be understandable by people without context on the specific changes. So usually we'll change the wording of the PR commits a little - e.g. this line would be something like "Make CloudWatch publishing failures visible by adding additional logging in <className>"

Reply:
addressing